.air.toml

```diff
@@ -41,4 +41,9 @@ tmp_dir = "tmp"
 
 [screen]
   clear_on_rebuild = false
-  keep_scroll = true
\ No newline at end of file
+  keep_scroll = true
+
+[proxy]
```

Comment:
Proxy helps with air port to backend port and also hot reloads on fronted changes.